### PR TITLE
Update reference links to stratum main branch

### PR DESCRIPTION
**Pull request by eth0xFEED**
Probably not needed until next tuesday when the github scripts used to update default branch to `main`

### CODE REVIEW

**Review by bocon13 — APPROVED** [review]
LGTM. Thanks!

**Comment by pudelkoM** [comment]
Also need to fix `.circleci/check-cpp-format.sh`

**Review by pudelkoM — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,4 +53,4 @@ Some additional points:
 ## Community Guidelines
 
 This project follows [Google's Open Source Community
-Guidelines](https://opensource.google.com/conduct/) and ONF's [Code of Conduct](https://github.com/stratum/stratum/blob/master/CODE_OF_CONDUCT.md).
+Guidelines](https://opensource.google.com/conduct/) and ONF's [Code of Conduct](https://github.com/stratum/stratum/blob/main/CODE_OF_CONDUCT.md).
```
Comment (pudelkoM) [inline CONTRIBUTING.md:56]: let's make this a relative link

**Review by pudelkoM — APPROVED** [review]

**Comment by pudelkoM** [comment]
Markdown links are a bit tricky:

- https URLs `[link](https://github.com/stratum/stratum/blob/main/some/file)`: depend on GH, hard code the branch, clickable on GH and most editors
- relative file link `[link](../some/file)`: clickable on GH and most editors (both raw md and preview), harder to update when files move?
- absolute file link `[link](/some/file)`: clickable on GH and most editors in raw mode, but not preview, easier to update?

**Comment by bocon13** [comment]
Thanks, @pudelkoM!

**Pull request merged**